sale_fixed_discount/readme/INSTALL.md

```diff
@@ -0,0 +1,3 @@
+This module depends on module 'account_invoice_fixed_discount',
+available in
+<https://github.com/OCA/account-invoicing/tree/16.0/account_invoice_fixed_discount>
```

Comment:
Change to 17.0

Reply:
Hi @JordiBForgeFlow , This changes has been updated to this commit : [2287d4d069bbc3b240bd48189efcbfde57beb4b9](https://github.com/OCA/sale-workflow/pull/2952/commits/2287d4d069bbc3b240bd48189efcbfde57beb4b9)
please review